modules/tinymce/CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The formatter can now apply a format to a `contenteditable="false"` element by wrapping it. Configurable using the `format_noneditable_selector` option. #TINY-8905
 - The autocompleter now supports a multiple character trigger using the new `trigger` configuration #TINY-8887
 
+### Changed
+- `@menubar-row-separator-color` variable now has no effect on the divider between Menubar and Toolbar and only controls the color of the separator lines drawn in multiline Menubar. `@toolbar-separator-color` variable is now exclusively used affects the color of separator between Menubar and Toolbar #TINY-8632
```

Comment:
Should this some how explain that the variable here is a skin variable? Also does it effect lines drawn in a multiline toolbar? I guess this log line only makes sense to people who know that these things are anyway. Maybe we could stick details into the release notes.

Reply:
> Also does it effect lines drawn in a multiline toolbar?

No changes here, it is still controlled via `@toolbar-row-separator-color`.

I believe [changes suggested by Lee](https://github.com/tinymce/tinymce/pull/7914/commits/2d9b6200e7fef3dcad2f1d66777d2c1373572cdf) address your concerns about the length and make the fact that variables are skin-related more clear.

Reply:
Yes, perfect.